src/main/java/de/tum/in/www1/artemis/repository/BuildJobRepository.java

```diff
@@ -103,4 +103,16 @@ Page<Long> findAllByFilterCriteria(@Param("buildStatus") BuildStatus buildStatus
             """)
     List<BuildJobResultCountDTO> getBuildJobsResultsStatistics(@Param("fromDateTime") ZonedDateTime fromDateTime, @Param("courseId") Long courseId);
 
+    /**
+     * Get the number of build jobs for exercise id.
+     */
+    @Query("""
+            SELECT COUNT(b)
+            FROM BuildJob b
+                LEFT JOIN Result r ON b.result.id = r.id
+                LEFT JOIN Participation p ON r.participation.id = p.id
+                LEFT JOIN Exercise e ON p.exercise.id = e.id
+            WHERE e.id = :exerciseId
+            """)
+    long countBuildJobsByExerciseId(@Param("exerciseId") Long exerciseId);
```

Comment:
The server style is complaining about this doc comment:

-  `/home/runner/work/Artemis/Artemis/src/main/java/de/tum/in/www1/artemis/repository/BuildJobRepository.java:109: @return tag should be present and have description. [JavadocMethod]`
- `/home/runner/work/Artemis/Artemis/src/main/java/de/tum/in/www1/artemis/repository/BuildJobRepository.java:117:63: Expected @param tag for 'exerciseId'. [JavadocMethod]`

Can you update the doc comment?